Altis_Life.Altis/core/functions/fn_playerSkins.sqf

```diff
@@ -12,35 +12,35 @@ switch (playerSide) do {
     case civilian: {
         if (LIFE_SETTINGS(getNumber,"civ_skins") isEqualTo 1) then {
             if (uniform player isEqualTo "U_C_Poloshirt_blue") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_1.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_1.paa"];
             };
             if (uniform player isEqualTo "U_C_Poloshirt_burgundy") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_2.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_2.paa"];
             };
             if (uniform player isEqualTo "U_C_Poloshirt_stripped") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_3.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_3.paa"];
             };
             if (uniform player isEqualTo "U_C_Poloshirt_tricolour") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_4.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_4.paa"];
             };
             if (uniform player isEqualTo "U_C_Poloshirt_salmon") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_5.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_5.paa"];
             };
             if (uniform player isEqualTo "U_C_Poloshirt_redwhite") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_6.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_6.paa"];
             };
             if (uniform player isEqualTo "U_C_Commoner1_1") then {
-                player setObjectTextureGlobal [0, "textures\civilian_uniform_7.jpg"];
+                player setObjectTextureGlobal [0, "textures\civilian_uniform_7.paa"];
```

Comment:
These files don't exist. Seems they didn't exist before either.

Reply:
please see further down - I have moved the original non-paa textures out of Altis_Life.Altis into a new folder and replaced them by PAA versions
conversion was done by TexView2 as the batch processing with ImageToPAA failed for some unknown reason

Reply:
That's because ImageToPAA only allows PNG > PAA, and not JPG > PAA